### DISCUSSION TITLE
How can I tell which dependency is going to get updated when looking at the Onboarding PR?

### DISCUSSION THREAD

**Discussion by janeklb** [renovatebot/renovate#23993] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'm in the process of onboarding a repo to use Renovate I can see that my `node` dependency is getting flagged for an update. The thing is, I have node versions referenced in a few places:
- base image (`FROM`) in `Dockerfile`
- `engines` section of `package.json`
- `.nvmrc`

Can renovate show the "dependency source" that it has detected and plans on updating?

### What you tried so far.

Nothing

### Relevant debug logs

N/A

**Comment by viceice** [comment, marked as answer]
no, but you find that information inside the debug logs. search for package files with updates

**Reply by rarkins** [reply]
https://developer.mend.io

**Reply by janeklb** [reply]
Thank you both 🙇
